documentation/site/layouts/ecf1-service-rules.html

```diff
@@ -19,12 +19,10 @@ <h1 class="govuk-heading-xl"><%= item[:title] %></h1>
               <%= yield %>
             </article>
           </div>
-          <div class="govuk-grid-column-one-third">
```

Comment:
could you explain why we removed this please, or why we changed the stylesheet?

Reply:
I think we should keep this as I can see it in the other ecf1 pages 🤔 not the ones we are editing but others have it

Reply:
Yeah sorry can we keep this for now, until we're able to archive off the old ECF1 rules.